### DISCUSSION TITLE
Solcast API ran out by 06:00 am

### DISCUSSION THREAD

**Discussion by screenagerbe** [nathanmarlor/foxess_em#264] [Ideas]
I've got 2 sites set up for one location (SW panels and NE panels) in Solcast.
Due to this setup I run out with API calls by 06:00.
![image](https://user-images.githubusercontent.com/82099983/232002893-ae15f942-2b02-47c3-98e0-f0834339a496.png)

Is there a way that the second call at 06:00 is scheduled later in the day, let's say 14:00 local time or so?

**Comment by nathanmarlor** [comment]
Unfortunately not - the integration will always schedule the first update at 6am then dynamically allocate the remaining across the day. With two sites and only 10 calls there isn't a lot to work with. Apparently Solcast will bump you back to 50 if you email them explaining what you're trying to achieve :)

**Reply by screenagerbe** [reply]
> Unfortunately not - the integration will always schedule the first update at 6am then dynamically allocate the remaining across the day. 

I've noticed I'm only using 4 API calls a day for the last two weeks and sometimes even 0 (no updates)
The only way to get it going again is going through the config menu after rebooting HA.
Any idea what's going on there?

**Comment by screenagerbe** [comment]
thx for the tip, I'll ask them nicely 😇

**Comment by screenagerbe** [comment]
no joy, looks like 10 api calls is the max these days.
Which, unfortunately, makes it rather unusable for use in automations

an example:
![image](https://github.com/nathanmarlor/foxess_em/assets/82099983/11a200e3-8cdf-4ef6-8e54-384b7d3a3578)
images left and middle are forecasts during the day, image on the right is the "forecast" day after
